INFO  | 03/31/2020 04:05:14 PM | File run_exp.py           | Line 26    | Writing log file to exp/GRAN\GRANMixtureBernoulli_transport_2020-Mar-31-16-05-13_3088\log_exp_3088.txt
INFO  | 03/31/2020 04:05:14 PM | File run_exp.py           | Line 27    | Exp instance id = 3088
INFO  | 03/31/2020 04:05:14 PM | File run_exp.py           | Line 28    | Exp comment = None
INFO  | 03/31/2020 04:05:14 PM | File run_exp.py           | Line 29    | Config =
INFO  | 03/31/2020 04:05:15 PM | File gran_runner.py       | Line 124   | Train/val/test = 8/2/2
INFO  | 03/31/2020 04:05:15 PM | File gran_runner.py       | Line 137   | No Edges vs. Edges in training set = 2.0
ERROR | 03/31/2020 04:05:18 PM | File run_exp.py           | Line 42    | Traceback (most recent call last):
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\run_exp.py", line 40, in main
    runner.test()
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\runner\gran_runner.py", line 310, in test
    A_tmp = model(input_dict)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\parallel\data_parallel.py", line 150, in forward
    return self.module(*inputs[0], **kwargs[0])
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\model\gran_mixture_bernoulli.py", line 480, in forward
    A = self._sampling(batch_size)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\model\gran_mixture_bernoulli.py", line 340, in _sampling
    node_pos = torch.cat((node_pos, (torch.ones(2,K)*np.inf)), dim = 1)
RuntimeError: Expected object of backend CUDA but got backend CPU for sequence element 1 in sequence argument at position #1 'tensors'

